tests/bearlib/languages/documentation/documentation_extraction_testdata/default.py

```diff
@@ -37,3 +37,30 @@ def best_docstring(param1, param2):
              Cut to the Next Line.
     """
     return None
+
+def docstring_find(filename):
+    """
+    This is dummy docstring find function!
+
+    :param filename:
+        contains filename
```

Comment:
In the pre-existing examples in this file , the param description is Sentence case.  Is that the 'default' style?
Your examples are inconsistent.  What should the default style be?